src/pages/index.js

```diff
@@ -85,9 +140,19 @@ viewImagePopup.setEventListeners();
 
 //edit profile popup
 const handleProfileEditSubmit = (inputValues) => {
-  userInfo.setUserInfo(inputValues.name, inputValues.about);
-
-  editProfilePopup.close();
+  editProfilePopup.renderLoading(true, "Saving...");
```

Comment:
Needs correction: When I update the profile info the description becomes `underfined`